daemon/container.go

```diff
@@ -833,6 +833,7 @@ func (container *Container) Exec(execConfig *execConfig) error {
 	// Exec should not return until the process is actually running
 	select {
 	case <-waitStart:
```

Comment:
I think we can use `execConfig.waitStart` here and also `close(execConfig.waitStart)` in the previous callback function https://github.com/mountkin/docker/blob/fix-exec-resize-panic/daemon/container.go#L826 and remove local variable `waitStart`?